app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarUiController.kt

```diff
@@ -153,11 +153,12 @@ class ToolbarUiController(
 
             layout.desktopModeButton.isChecked = it.desktopModeChecked
             layout.turboButton.isChecked = it.turboChecked
-            layout.turboButton.contentDescription = layout.context.resources.getString(
-                // TODO known issue: updating the content description doesn't update the hanger until focus navigates away and back again. fix this
+            val turboText = layout.context.resources.getString(
                 turboButtonContent.contentDescriptionId,
                 if (it.turboChecked) "ON" else "OFF"
```

Comment:
This "ON" and "OFF" would need to be localized.

Reply:
🤦‍♂ 
I don't know how I missed that.  Thank you for catching it.